Documentation/ceph-cluster-crd.md

```diff
@@ -356,6 +356,32 @@ The following storage selection settings are specific to Ceph and do not apply t
 * Mimic 13.2.3 or newer
 * Nautilus
 
+### Storage Selection Via Ceph Drive Groups
+
+Ceph supports adding devices as OSDs by Ceph Drive Group definitions in later versions of Ceph
```

Comment:
> I think this fits that description still. I need to note here that Rook overrides Ceph's definition of placement, service_id, and host_pattern so that users know to expect that Rook controls those spec items and that anything they specify will be overridden by Rook.

Let's assume that when someone comes to the Rook project, they start with Rook docs and they have never heard of Ceph drive groups. So it would be helpful to focus the Rook docs on what scenarios they can accomplish with drive groups, give some examples of where they are needed, etc. To contrast, this section sounds like it assumes the user knows what Ceph drive groups are and the entire section focuses on how rook drive groups are different from ceph's documentation. Stating these differences is a good thing, it would just help if first the doc gave examples of use cases from Rook's perspective. Without this, we force them to go to the Ceph docs just to understand why they would use drive groups beyond the sentence `Ceph Drive Groups allow for specifying highly advanced OSD layouts on nodes
including non-homogeneous nodes.`

Reply:
I understand now! I added a paragraph explanation.